CHANGELOG.md

```diff
@@ -49,6 +51,7 @@
   - ⚠️ `column.getCellActions`
     - Check [#1845](https://github.com/adazzle/react-data-grid/pull/1845) on how to migrate
   - ⚠️ `column.getRowMetaData`
+  - ⚠️ `column.filterable`
```

Comment:
```suggestion
```
It's redundant now.

Reply:
Let's keep it for now as one is column prop and other is grid prop